social_core/utils.py

```diff
@@ -10,7 +10,7 @@
 
 import requests
 from requests.adapters import HTTPAdapter
-from requests.packages.urllib3.poolmanager import PoolManager
+from urllib3.poolmanager import PoolManager
```

Comment:
Maybe this is what breaks the testsuite? At least I don't see anything else what could be suspicious. Looking at the code I wonder if SSLHttpAdapter is still needed these days. I've tried to remove it in https://github.com/python-social-auth/social-core/pull/987.

Reply:
No... I just accidentally left in a test assertion. Fixing...